translations/dutch.json

```
@@ -500,6 +500,134 @@
+		"Note that rooms can set their own language, which will override this setting.": "",
+		"/updatesettings expects JSON encoded object.": "",
+		"Unable to parse settings in /updatesettings!": "",
+		"Must be in a battle.": "",
+		"User ${target} not found.": "",
+		"Must be a player in this battle.": "",
+		"${targetUser.name} has not requested extraction.": "",
+		"You have already consented to extraction with ${targetUser.name}.": "",
+		"${user.name} consents to sharing battle team and choices with ${targetUser.name}.": "",
+		"No input log found.": "",
+		"${targetUser.name} has extracted the battle input log.": "",
+		"This command only works in battle rooms.": "",
+		"This command only works when the battle has ended - if the battle has stalled, use /offertie.": "",
+		"Alternatively, you can end the battle with /forcetie.": "",
+		"${user.name} has extracted the battle input log.": "",
+		"You already extracted the battle input log.": "",
+		"Battle input log re-requested.": "",
+		"Invalid input log.": "",
+		"Your input log contains untrusted code - you must have console access to use it.": "",
+		"This command can only be used in a battle.": "",
+		"Only players can extract their team.": "",
+		"Use a number between 1-6 to view a specific set.": "",
+		"The Pokemon \"${target}\" is not in your team.": "",
+		"That Pokemon is not in your team.": "",
+		"View team": "",
+		"Must be in a battle room.": "",
+		"This server does not allow offering ties.": "",
+		"You can't offer ties in tournaments.": "",
+		"It's too early to tie, please play until turn 100.": "",
+		"No other player is requesting a tie right now. It was probably canceled.": "",
+		"${user.name} is offering a tie.": "",
+		"Accept tie": "",
+		"Reject": "",
+		"Must be a player to accept ties.": "",
+		"You have already agreed to a tie.": "",
+		"${user.name} accepted the tie.": "",
+		"All players have accepted the tie.": "",
+		"Must be a player to reject ties.": "",
+		"${user.name} rejected the tie.": "",
+		"This room doesn't have an active game.": "",
+		"This kind of game can't be forfeited.": "",
+		"This game doesn't support /choose": "",
+		"This game doesn't support /undo": "",
+		"You can only save replays for battles.": "",
+		"This battle can't have hidden replays, because the tournament is set to be forced public.": "",
+		"The replay for this battle is already set to hidden.": "",
+		"${user.name} hid the replay of this battle.": "",
+		"You can only do this in battle rooms.": "",
+		"You can only add a Player to unrated battles.": "",
+		"Player must be set to \"p1\" or \"p2\", not \"${target}\".": "",
+		"User ${name} not found.": "",
+		"User ${name} must be in the battle room already.": "",
+		"This room already has a player in slot ${target}.": "",
+		"${targetUser.name} is already a player in this battle.": "",
+		"${name} was added to the battle as Player ${target.slice(1)} by ${user.name}.": "",
```

```suggestion
		"${name} was added to the battle as Player ${target.slice(1)} by ${user.name}.": "",
```

I would pull `target.slice(1)` out into `playerNum`